scripts/github/sync-issues-to-project.sh

```
@@ -0,0 +1,369 @@
+                shift 2
+                ;;
+            --limit)
+                LIMIT_COUNT="$2"
+                shift 2
+                ;;
+            --check-orphaned)
+                CHECK_ORPHANED=true
+                shift
+                ;;
+            --remove-orphaned)
+                REMOVE_ORPHANED=true
```

Shellcheck (0.10.0)

[warning] REMOVE_ORPHANED appears unused. Verify use (or export if used externally).

(SC2034)